config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "monitoringCluster": "mad",
+  "logDirectory": "logs",
+  "pipelinesDirectory": "/opt/mad/config/pipelines",
+  "httpHost": "0.0.0.0",
+  "httpPort": 7090,
+  "httpHealthCheckPath": "/ping",
+  "httpStatusPath": "/status.json",
+  "jvmMetricsCollectionInterval": "PT.5S",
+  "akkaConfiguration": {
+    "akka": {
+      "loggers": [
+        "akka.event.slf4j.Slf4jLogger"
+      ],
+      "loglevel": "DEBUG",
```

Comment:
This is the minimum level that using the Akka logging will send across to the logging actor.  If this is set above DEBUG, you will never get DEBUG messages sent to slf4j.  Make sense?